R/glmfun.R

```diff
@@ -16,7 +16,7 @@ standardization <- function(x, center = TRUE, scale = TRUE, weights = NULL) {
     mx <- rep(0, ncol(x))
   }
   if (scale) {
-    sx <- apply(x, 2, weighted.sd, w)
+    sx <- apply(x, 2, .weighted_sd, w)
```

Comment:
See my comment concerning `.select()` for why I personally prefer not to start function names with a dot.

Reply:
Based on the original name, I was certain it's from some other package and googling also reveals `weighted.sd` already exists in another CRAN package. I've liked how in other Stan related packages dot makes it easy to see it's an internal function